packages/slate/src/interfaces/editor.ts

```diff
@@ -1072,15 +1073,15 @@ export const Editor: EditorInterface = {
         }
       }
 
-      const dirtyPaths = getDirtyPaths(editor)
-
+      const initialDirtyPathsLength = getDirtyPaths(editor).length
       let iteration = 0
 
       while (getDirtyPaths(editor).length !== 0) {
         if (
           !editor.shouldNormalize({
-            iteration,
             dirtyPaths: getDirtyPaths(editor),
```

Comment:
Not sure what the performance overhead of `getDIrtyPaths` is, but it's not used by `shouldNormalize` and passing it here means it's done twice per iteration.